nifi-nar-bundles/pom.xml

```diff
@@ -59,6 +59,7 @@
         <module>nifi-jms-bundle</module>
         <module>nifi-cassandra-bundle</module>
         <module>nifi-spring-bundle</module>
+        <module>nifi-alluxio-bundle</module>
```

Comment:
Will also need to provide the alluxio-nar in the nifi-assembly pom and as a managed dependency in the root pom.